packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -1225,20 +1225,25 @@ const ArrayMixin = Mixin.create(Enumerable, {
   },
 
   /**
-    Returns `true` if the passed object can be found in the array.
+    Used to determine if the array contains the passed object.
+    Returns `true` if found, `false` otherwise.
+
+    The optional `startAt` argument can be used to pass a starting
+    index to search from, effectively slicing the searchable portion 
+    of the array. If it's negative it will add the array length to
+    the startAt value passed in as the index to search from. If less
+    than or equal to `-1 * array.length` the entire array is searched.
+
     This method is a Polyfill for ES 2016 Array.includes.
-    If no `startAt` argument is given, the starting location to
-    search is 0. If it's negative, searches from the index of
-    `this.length + startAt` by asc.
 
     ```javascript
     [1, 2, 3].includes(2);     // true
     [1, 2, 3].includes(4);     // false
     [1, 2, 3].includes(3, 2);  // true
     [1, 2, 3].includes(3, 3);  // false
-    [1, 2, 3].includes(3, -1); // true
-    [1, 2, 3].includes(1, -1); // false
-    [1, 2, 3].includes(1, -4); // true
+    [1, 2, 3].includes(3, -1); // true, equivalent to includes(3, 2)
```

Comment:
I'm somewhat confused by the comments in here. I would suggest not including them, or having separate example blocks with an explanation included.